### DISCUSSION TITLE
Can't find the .exe

### DISCUSSION THREAD

**Discussion by Parma-John** [philosowaffle/peloton-to-garmin#261] [Q&A]
<img width="578" alt="Screenshot 2022-06-02 195312" src="https://user-images.githubusercontent.com/106790854/171777454-af055ea7-7c4f-465e-996a-ef4c8a361752.png">

I'm on windows and I don't have a .exe file. Did I miss something or download the wrong file? I'm very new to this but I tried to include a pic of where I thought the .exe file was supposed to be. Any help would be appreciated!

**Comment by Parma-John** [comment]
So I found it but when run the application it instantly disappears is that normal? and how long does it take to upload?

**Reply by Parma-John** [reply]
Also has anyone tried with Windows 11? I didn't realize my computer updated to it

**Reply by philosowaffle** [reply]
To clarify, did you follow the steps [documented here](https://philosowaffle.github.io/peloton-to-garmin/install/windows.html), especially the part about updating your configuration file before running the application?

Additionally, if any logs were generated they should be somewhere in the directory where you launched the exe from, possibly in a folder called `output`.  Those logs could be helpful in diagnosing the problem.

I have not tested on Windows 11, but I would be very surprised if there were any incompatibility issues.

**Reply by Parma-John** [reply]
Yeah, I figured it out, sorry for all the questions. Just took some troubleshooting on my end. uploading it first helped a lot. and I can confirm that it works on Windows 11. Thanks again!
